options.go

```diff
@@ -61,28 +62,17 @@ func WithRequestsType(requestsType string) Option {
 // Warning: WithConnectionString will be removed at next major release
 // (connection string will be required string param of ydb.Open)
 func WithConnectionString(connectionString string) Option {
+	if connectionString == "" {
+		return nil
+	}
 	return func(ctx context.Context, c *connection) error {
```

Comment:
```suggestion
	return func(ctx context.Context, c *connection) error {
		if connectionString == "" {
			return nil
		}
```

И не надо пропуски nil делать в опциях